### DISCUSSION TITLE
Idea!

### DISCUSSION THREAD

**Discussion by ghost** [zer0Kerbal/Taerobee#111] [Ideas]
[There's an "older" mod, Wernher's Old Bits, that hasn't been updated since 1.11.0 and it has a highly detailed V-2 Engine.](https://spacedock.info/mod/42/Wernher%27s%20Old%20Bits)

Perhaps could we integrate it into Taerobee?

![Ew3fVfx](https://user-images.githubusercontent.com/74610801/151822938-287e82f9-5e98-4b1c-b24e-622ae21ea9d9.png)

**Comment by zer0Kerbal** [comment]
Interesting idea @JamesErvin-5 , I will ask the author - we are on good terms. He might allow total adoption of all. 50+ parts. Shouldn't get your hopes up. News when there is news.

**Reply by ghost** [reply]
"News when there is news."
That's probably the most KSP Mod Dev saying I've ever heard lol

**Reply by zer0Kerbal** [reply]
@[AtomikkuSan](https://github.com/AtomikkuSan)
> "News when there is news."
> That's probably the most KSP Mod Dev saying I've ever heard lol
@AtomikkuSan, I've asked the author - who is currently very active. Will see about outcome.
